apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -664,6 +664,13 @@ const AvailabilityPage = ({ profile, eventType }: Props) => {
                   )*/}
               </div>
               <SlotPicker
+                weekStart={
+                  typeof profile.weekStart === "string"
+                    ? (["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"].indexOf(
+                        profile.weekStart
+                      ) as 0 | 1 | 2 | 3 | 4 | 5 | 6)
+                    : profile.weekStart /* Allows providing weekStart as number */
```

Comment:
We really should store weekStart as a number in the DB (Or DB type) so this will allow that backwards compatibility when we get to it.